light-rest-4j/src/main/resources/templates/rest/openapi/config.rocker.raw

```diff
@@ -0,0 +1,18 @@
+#----------------------------------------------------------------------------------------------------------------
+# Scalable Config file
+#
+# This file serves as a configuration extension platform. Functions are list below:
+#
+# [1]   exclusionConfigFileList: this configuration will be used by the light-4j/config module, when reading
+#       config files. it allows the listing of files which will be excluded from parameterized values set at
+#       the command-line or in a values.yml file
+#       Notes:
+#       File name included in the list will be excluded
+#       If the file is not provided, the config module will safely ignore it
+#----------------------------------------------------------------------------------------------------------------
+exclusionConfigFileList:
+	- openapi.yaml
+	- openapi.json
+	- openapi.yml
+	- values.yml
+	- status.yml
```

Comment:
Could we also add the other options for values & status? Like values.yaml, values.json, status.yaml, & status.json?